packages/dashboard/src/initialization/Dashboard.tsx

```diff
@@ -60,8 +60,8 @@ export default function DashboardRoot() {
                                         <SharedContextProvider>
                                             <HashRouter>
                                                 <Pages />
+                                                <PluginRender />
                                             </HashRouter>
-                                            <PluginRender />
```

Comment:
Why?

Reply:
please comment https://github.com/DimensionDev/Maskbook/pull/5986 in this pr, here just merge and use some component in that pr